src/OneMap.js

```diff
@@ -4,7 +4,7 @@ require('./style.css');
 const objectAssign = require('object-assign');
 const dom = require('./utils/dom');
 
-class Map {
+class OneMap {
```

Comment:
What about using `AbstractMap` ? 

Using `OneMap` is confusing with https://github.com/Leadformance/oneMapToRuleThemAll/pull/10/files#diff-7aa54e60917997fcb386cdc0d478e090R125

Reply:
Yes, why not, I will check if there is not side effect.